### ISSUE TITLE
"Copy path" в контекстном меню для файлов 

### ISSUE THREAD

**Issue by SM-Sergey** [miranda-ng/miranda-ng#3602]
Хотелось бы иметь возможность четко скопировать только путь к скачанному оффлайн файлу в клипборд, а также тоже самое для давно скачанных файлов, еще до появления этой возможности на серверах протоколов. И во встроенных журналах, и в ньюстори.
Сейчас это не работает унифицировано - для старых файлов можно путь скопировать через простое "Copy" в ньюстори, но с излишками - первой строкой будет автор сообщения, дата и т.п.. Для новых - никак вообше

**Comment by dartraiden** [comment]
В NewStory с этим что-то странное:

1) "Copy URL" отсутствует
2) Присутствует "Copy file name"
3) Если сделать  "Copy file name" для отправленного файла, то копируется локальный путь к нему
Если сделать "Copy file name" для прнятого файла, то между путём и именем файла вставляется лишний слеш

`C:\Users\dartraiden\AppData\Roaming\Miranda NG\home\My received files\dartraiden2@jabber.ru-dartraiden2\\netlog.txt`

**Comment by dartraiden** [comment]
В IEView пункт "Копировать ссылку" недоступен (disabled)

**Comment by georgehazan** [comment]
Всё там норм с фокусом, проблема не в этом
И вообще тикет о том, что надо путь вырабатывать для ICQ каждый раз, а это я починил

**Comment by SM-Sergey** [comment]
> И вообще тикет о том, что надо путь вырабатывать для ICQ каждый раз, а это я починил
не-не, этот тикет про другое! Про возможность скопировать в клипборд не URL, а путь к файлу, где он лежит скачанный в локальной файловой системе.

**Comment by georgehazan** [comment]
Пункт Copy file name тоже представлен для скачанных файлов

**Comment by SM-Sergey** [comment]
Полупредставлен... Когда я создавал тикет - там было "для ньюстори и встроенных журналов" (да и сейчас осталось) - ну и я упорно искал это меню в журнале табсов. В ньюстори все ОК. Так что, простите, я не изучил коммиты, чтобы понять, где именно это сделано. Раз тикет закрыли, я подумал, что и там, и тут.

**Comment by dartraiden** [comment]
Да, только в NewStory есть смена Copy path/Copy URL. В прочих журналах только Copy URL.
